package.json

```
@@ -59,7 +59,7 @@
     "@lit-labs/ssr-dom-shim": "^1.2.0"
   },
   "peerDependencies": {
-    "theoplayer": "^6"
+    "theoplayer": "^7"
```

We don't want to do a breaking change for this.
```suggestion
    "theoplayer": "^6 || ^7"
```

---

Okay, maybe we do want to break for https://github.com/THEOplayer/web-ui/pull/58... 🤔